drivers/spi/spi_bitbang.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (c) 2021 Marc Reilly - Creative Product Design
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_spi_bitbang
```

Comment:
Is this the intent of the zephyr vendor prefix? There is another very similar driver with the name gpio_i2c. Should this be named gpio_spi to be consistent?

https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/i2c/i2c_gpio.c

Reply:
It made sense at the time :) but TBH I'm not sure.

There is also the "zephyr,2ic-emul" which has `#define DT_DRV_COMPAT zephyr_i2c_emul_controller` ... having a brief look through the binding and driver files, I get the vibe that the newer drivers use 'zephyr' as the vendor prefix, and the gpio_i2c and gpio_led etc do it a different way because they are older than the existing convention but remain the same for compatibility.

I don't really care though .. if there is a consensus that some other way is better I can change it.

Reply:
I think `zephyr` is an OK vendor prefix here following the rules laid out in https://docs.zephyrproject.org/latest/guides/dts/bindings.html#vendor-prefixes. But maybe @galak disagrees.